restaurants/src/app/components/restaurants/restaurants.component.html

```diff
@@ -1,9 +1,10 @@
 <h2>Restaurants</h2>
 <ol class="restaurants">
     <li *ngFor="let restaurant of restaurants | keyvalue">
-        <button [class.selected]="restaurant.key === selectedRestaurant" type="button" (click)="onSelect(restaurant.key)">
-        <span class="name">{{restaurant.value.name}}</span>
-        </button>
+    <button [class.selected]="restaurant.key === selectedRestaurant" type="button" (click)="onSelect(restaurant.key)">
+        <a routerLink ="/restaurant/{{restaurant.key}}">
+          {{restaurant.value.name}}</a>
+    </button>
     </li>
   </ol>
```

Comment:
Looks like indentation of the html above might be off? start of <ol> and end of </ol> aren't aligned... some of the elements inside are also misaligned.

Reply:
done